app/Cron/Hourly/DeletePireps.php

```diff
@@ -38,7 +39,10 @@ public function handle(CronHourly $event): void
     protected function deletePireps(int $expire_time_hours, int $state)
     {
         $dt = Carbon::now('UTC')->subHours($expire_time_hours);
-        $pireps = Pirep::where('created_at', '<', $dt)->where(['state' => $state])->get();
+        $pireps = Pirep::where('created_at', '<', $dt)
+            ->where(['state' => $state])
+            ->where(['status', '<>', PirepStatus::PAUSED])
```

Comment:
`->where('status', '<>', PirepStatus::PAUSED)` should work.

Reply:
Good catch